client/core/rs/src/api/execute/mod.rs

```diff
@@ -152,6 +152,9 @@ pub enum Execution {
   StopStack(StopStack),
   DestroyStack(DestroyStack),
   BatchDestroyStack(BatchDestroyStack),
+  /// Runs a one-time command against a service using `docker compose run`.
+  #[clap(alias = "run")]
```

Comment:
I think this alias is too generic unfortunately.

However for common stuff geared towards stacks, I want to make dedicated `km` command for this

```
#[clap(alias = "compose")]
Stack(stack::Stack)
```

So ultimately this could be more conveniently called at `km compose run ...` in place of the top level Execute alias.

Reply:
Makes sense, I kind of felt the same. I can just kill the alias for now